sql/index/pilosa/driver.go

```diff
@@ -116,6 +105,37 @@ func (d *Driver) LoadAll(db, table string) ([]sql.Index, error) {
 	return indexes, err
 }
 
+var errCorruptIndex = errors.NewKind("the index in %q is corrupt")
+
+func (d *Driver) loadIndex(path string) (sql.Index, error) {
+	ok, err := index.ExistsProcessingFile(path)
+	if err != nil {
+		return nil, err
+	}
+
+	if ok {
+		log := logrus.WithFields(logrus.Fields{
+			"err":  err,
+			"path": path,
+		})
+		log.Warn("could not read index file, index is corrupt and will be deleted")
+
+		if err := os.RemoveAll(path); err != nil {
```

Comment:
So when the file exists we just delete it, not checking how long it exists?
it means that we can save and load, over and over

Reply:
This is only used when the app starts